app/workers/daily_digest_initiator_worker.rb

```diff
@@ -1,5 +1,13 @@
 class DailyDigestInitiatorWorker < DigestInitiatorWorker
-  def perform(date = Date.current)
-    DigestInitiatorService.call(date: date, range: Frequency::DAILY)
+  sidekiq_options lock: :until_executed,
```

Comment:
Working on something different I noticed sidekiq_options are already set in DigestInitiatorWorker and so figured this might override the other options (not sure if they merge or not) might be good to consolidate either repeating them all setting them all in the parent class either way.

Reply:
Another good find - you're both on a roll today. I think it's clearer to have everything in one place, so I've added another commit to make this so.